jekyll/_cci2/certificates.md

```diff
@@ -7,13 +7,35 @@ categories: [administration]
 order: 55
 ---
 
-This document describes the process for using an ELB with HTTPS/SSL enabled.  You will need certificates for the ELB and CircleCI Server, covered in the following sections:
+This document provides a script for using a custom Root Certifcate Authority and the process for using an Elastic Load Balancing certificate in the following sections:  
 
 * TOC
 {:toc}
 
+
+## Using a Custom Root CA
+
+This section describes how to use a custom Root Certificate Authority (CA).
+
+Some installation environments use internal Root Certificate Authorities for encrypting and establishing trust between servers.  If you are using a Root certificate, you will need to import and mark it as a trusted certificate at CircleCI GitHub Enterprise instances. CircleCI will respect such trust when communicating with GitHub and webhook API calls.
+
+CA Certificates must be in a format understood by Java Keystore, and include the entire chain.
+
+The following script provides the necessary steps.
+
+```
+GHE_DOMAIN=github.example.com
```

Comment:
@mqarty - can we get someone who knows picard-dispatcher/build-agent to confirm that this will work for Docker registries using internal CAs?

Otherwise customer with a custom CA registry get an error pulling down 
>  Error pulling image reg.detss.xyz.net/caas/busybox:1.27: Error response from daemon: Get https://reg.detss.xyz.net/v1/_ping: x509: certificate signed by unknown authority... retrying
  image cache not found on this host, downloading reg.detss.xyz.net/caas/busybox:1.27

Reply:
@junejung or @MikeyYeahYeah can you help out here?